### PR TITLE
CMRARC-825: Add handling of number array

### PR DESCRIPTION
**Pull request by htranho**

### CODE REVIEW

File: app/helpers/cmr_helper.rb [hunk 1]
```diff
@@ -37,12 +36,13 @@ def flatten_to_array(collection_hash, parent_string = "")
           new_collection_arr.concat(flattened_sub_array)
         elsif sub_value.is_a?(Array)
           #some keyword groupings are presented as lists of strings, do not want to seperate these
-          if sub_value[0].is_a?(String)
+          if ( sub_value[0].is_a?(String) || sub_value[0].is_a?(Numeric))
```
Comment (cgokey) [inline app/helpers/cmr_helper.rb:39]: something change with CMR?

Comment (cgokey) [reply app/helpers/cmr_helper.rb:39]: maybe add a comment to the code of what changed

Comment (htranho) [reply app/helpers/cmr_helper.rb:39]: Maybe array of numbers is new for this code.

Comment (cgokey) [reply app/helpers/cmr_helper.rb:39]: Just curious, what field was it parsing that had an array of numbers?

**Review by dmistry1 — APPROVED** [review]

**Review by cgokey — APPROVED** [review]

**Pull request merged**